### PR TITLE
Readline -> 8.1

### PR DESCRIPTION
**Pull request by satmandu**
Compiles/Packages properly:
- [x] x86_64
- [x] i686
- [x] armv7l

I won't be so bold as to say it works splendidly,  but I'm seeing zero issues on x86_64.

### CODE REVIEW

File: packages/readline.rb [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-static',
-           '--with-curses'
+    system "./configure #{CREW_OPTIONS} \
+    --disable-install-examples \
+    --with-curses"
```
Comment (uberhacker) [inline packages/readline.rb:27]: Does this break anything?  There are so many packages that depend on this.

Comment (satmandu) [reply packages/readline.rb:27]: Better safe than sorry. I'll take that line out.

Comment (uberhacker) [reply packages/readline.rb:27]: Probably a better solution would be to recompile other packages if something breaks.  More work, I know but we don't want to sacrifice capabilities, right?

Comment (satmandu) [reply packages/readline.rb:28]: yup. Updated with new uploads.

**Pull request merged**